spec/models/school_can_order_devices_notifications_spec.rb

```diff
@@ -1,26 +1,17 @@
 require 'rails_helper'
+require 'shared/school_creator'
```

Comment:
`spec/rails_helper.rb` has the line 
```ruby
Dir[Rails.root.join('spec/support/**/*.rb')].sort.each { |f| require f }
```
If you we move this file to there is should get loaded before the suite runs and you won't need to explicitly require this file every time it's needed

Reply:
Good idea.